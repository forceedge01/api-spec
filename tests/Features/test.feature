Feature:
    Given I have an api
    As a developer
    I want to test it

    Scenario:
        Given I set the following headers:
            | content-type | application/json |
            | accept       | en               |
        When I make a POST request to "User" endpoint with body:
            """

            """
        Then I expect a 200 "User" response

    Scenario:
        When I make a GET request to "User" endpoint with query string "exception=1"
        Then I expect a 500 "User" response

    Scenario:
        When I make a GET request to "User" endpoint with query string "test=1"
        Then I expect a 201 "User" response
