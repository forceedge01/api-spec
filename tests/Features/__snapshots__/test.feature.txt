a:4:{s:17:"200-user-response";s:26:"{"success":true,"id":2233}";s:22:"500-exception-response";s:53:"{"success":false,"error":"Something went wrongsssss"}";s:15:"201-get-request";s:82:"{"success":true,"msg":"Record created","id":"887987fa89s7df87as9d7f87asdf798asdf"}";s:32:"200-post-request-to-create-user.";s:26:"{"success":true,"id":2233}";}